components/list/list.js

```diff
@@ -16,6 +16,11 @@ class List extends SelectionMixin(LitElement) {
 
 	static get properties() {
 		return {
+			/**
+			 * Whether to the user can drag multiple items
+			 * @type {boolean}
+			 */
+			dragMultiple: { type: Boolean, attribute: 'drag-multiple' },
```

Comment:
Consumers will need to opt into multi drag & drop since their code will need to handle it.